tsconfig.json

```diff
@@ -7,8 +7,10 @@
     "skipLibCheck": true,
     "allowSyntheticDefaultImports": true,
     "moduleResolution": "node",
-    "lib": ["es2015"],
-    "skipLibCheck": true
+    "lib": ["es6", "es2015"],
```

Comment:
Those are actually the same, let's leave es6 here.

Reply:
fixed 5538f62